.circleci/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+nvcc_compiler:
+  - nvcc
+nvcc_compiler_version:
+  - 10.0
```

Comment:
Same story here.

Reply:
This is handled by `conda-smithy` and the right `conda_build_config.yaml` fields now, right?

Reply:
Fixed!

Reply:
> This is handled by `conda-smithy` and the right `conda_build_config.yaml` fields now, right?

Yeah that's right.